documentation/vocs.config.tsx

```diff
@@ -190,7 +190,7 @@ export default defineConfig({
       items: [
         { text: "From A Foundry Project - coming soon" },
         { text: "From A Hardhat Project - coming soon" },
-        { text: "Using Reth ExExes - coming soon" },
+        { text: "Using Reth ExExes", link: "/docs/advanced/using-reth-exex" },
```

Comment:
so i would say now this is a good standalone guide but the standalone guide should make it super easy for people to get up and running alongside we should have callouts in the https://rindexer.xyz/docs/start-building/create-new-project that --reth flag exists etc dont want this hidden away

Reply:
Makes sense!